cime/config/e3sm/machines/config_compilers.xml

```diff
@@ -1987,7 +1994,6 @@ ntel/x86_64/2013/composer_xe_2013/composer_xe_2013_sp1.3.174/mkl/include </base>
   <MPI_LIB_NAME> mpich</MPI_LIB_NAME>
   <MPI_PATH>/usr/tce/packages/mvapich2/mvapich2-2.2-intel-18.0.1/</MPI_PATH>
   <NETCDF_PATH>/usr/tce/packages/netcdf-fortran/netcdf-fortran-4.4.4-intel-18.0.1/</NETCDF_PATH>
-  <PNETCDF_PATH>$ENV{PNETCDFROOT}</PNETCDF_PATH>
```

Comment:
I'm not sure but we may need to keep this line.  From what I can recall E3SM and SCREAM master both compile just fine on syrah, but just in case, if after this upstream merge scream won't compile on syrah out of the box then we can simply add this line back in as the first step towards looking for a fix.